src/fragments/lib/troubleshooting/common/upgrading.mdx

```diff
@@ -46,83 +46,70 @@ In order to use Amplify JavaScript V6, you will need to make sure you are using
 
 ## Step 2: Upgrade your Amplify project dependencies
 
-<Callout>
-  Make sure to delete your package-lock.json and node_modules folder before
-  running `npm install`
-</Callout>
+<Callout>Make sure to delete your package-lock.json and node_modules folder before running `npm install`</Callout>
 
 <BlockSwitcher>
+  <Block name="Web">
+    Upgrade to the latest Amplify library using the following command.
 
-<Block name="Web">
-
-Upgrade to the latest Amplify library using the following command.
-
-```
-npm install aws-amplify
-```
-
-The `aws-amplify` package in v6 includes the categories below:
-
-- Auth
-- API
-- Storage
-- Analytics
-- DataStore
-- In-App Messaging
-
-If you would like to use Geo, Predictions, PubSub, or Interactions you will need to install those packages separately. (see [category-specific migration instructions](/[platform]/build-a-backend/troubleshooting/migrate-from-javascript-v5-to-v6/#4-upgrade-category-usage))
-
-</Block>
-
-<Block name="React Native">
-
-In v6, you will no longer need the `amazon-cognito-identity-js`, so that dependency should be removed from your package.json.
+    ```
+    npm install aws-amplify
+    ```
 
-Then upgrade/install the necessary dependencies using the following command:
+    You can also upgrade by making sure the version of `aws-amplify` in your package.json file is `latest` or `^6.0.1` and re-installing dependencies by running `npm install`.
```

Comment:
Not sure if "also" makes sense here. If a customer does not change the Amplify dependency version then whatever the current version they are pointing at would be installed?

Reply:
Actually no, if they use the above script `npm install aws-amplify` it will upgrade to the latest version. At least from my experimentation that is accurate.